packages/gatsby-plugin-nginx/src/nginx-generator.ts

```diff
@@ -217,7 +217,7 @@ export function convertToRegExp(path: string) {
     // allows '<path>/*' to serve '<path>' (without trailing slash)
     // this is necessary because we are now removing trailing slashes from incoming requests
     // so exact matches can work when there is a trailing slash
-    converted = converted.slice(0, -catchAll.length) + '(?:/(.*))?'
+    converted = `${converted.slice(0, -catchAll.length)}(?:/(.*))?`
```

Comment:
this change introduces a Github Actions / Lint warning.
<img width="501" alt="Screen Shot 2022-03-22 at 11 46 01" src="https://user-images.githubusercontent.com/11325562/159509070-9fe5cb86-7120-4b4e-9d51-251936f0dfaa.png">

Reply:
I think he was trying to fix [the previous warning](https://github.com/vtex/faststore/commit/db6f1e173ec43747bf9cfa244b81d86db8a25ce3#diff-ce3b1e791d013aac1413693a4a6e5e04d40451196307f313d8b89a3740cbcf03R220) on this same line but got a different warning this time.

The rule is [prefer-template](https://eslint.org/docs/rules/prefer-template) and the changes make sense on my mind, so I'm not sure why it's still warning.